core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -589,10 +589,11 @@ private[spark] class BlockManager(
   private def doGetRemote(blockId: BlockId, asBlockResult: Boolean): Option[Any] = {
     require(blockId != null, "BlockId is null")
     val locations = Random.shuffle(master.getLocations(blockId))
+    val timeout = conf.getInt("spark.storage.fetchBlockTimeout", 1000) // seconds
```

Comment:
We now have support for retrieving properties specified like "1000s", so it'd be best to use that.

Rather than proliferating so many different timeout values, how about using `spark.network.timeout`? at least, this should fall back on that like others do.